### PR TITLE
Upgrade to Laravel 6.0

### PR DESCRIPTION
**Pull request by nicolocaruso85**

### CODE REVIEW

**Comment by nicolocaruso85** [comment]
fixes https://github.com/opendialogai/core/issues/208

File: composer.json [hunk 1]
```diff
@@ -14,25 +14,27 @@
...
   "require": {
-    "php": "^7.1.3",
-    "symfony/yaml": "^4.2",
+    "php": "^7.3.0",
+    "symfony/yaml": "^4.3",
     "php-ds/php-ds": "~1.2.",
     "guzzlehttp/guzzle": "~6.3",
     "justinrainbow/json-schema": "^5.2",
-    "spatie/laravel-activitylog": "^3.2",
+    "laravel/framework": "^6.0",
```
Comment (stu-greenshoots) [inline composer.json]: We do not need to list Laravel as a dependency, it comes with orchestra/testbench.

File: composer.json [hunk 2]
```diff
@@ -14,25 +14,27 @@
...
     "doctrine/dbal": "^2.9"
   },
   "config": {
     "preferred-install": "source"
   },
   "require-dev": {
-    "phpunit/phpunit": "^7.5",
+    "phpunit/phpunit": "^8.3",
     "mockery/mockery": "^1.2",
     "orchestra/testbench": "^3.8",
```
Comment (stu-greenshoots) [inline composer.json]: We need to update orchestra/testbench to 4.0

File: composer.json [hunk 3]
```diff
@@ -14,25 +14,27 @@
...
     "squizlabs/php_codesniffer": "^3.4",
     "barryvdh/laravel-ide-helper": "^2.6",
-    "matthewbdaly/artisan-standalone": "^0.0.8"
+    "matthewbdaly/artisan-standalone": "^0.0.8",
+    "dms/phpunit-arraysubset-asserts": "^0.1.0"
```
Comment (stu-greenshoots) [inline composer.json]: What is this used for?

**Comment by stu-greenshoots** [comment]
@nicolocaruso85 could you please review the comments and fix the merge conflict

**Pull request merged**